### PR TITLE
Fix Chunks in extensions

### LINKED ISSUE #7329: amp-bind console error
After the latest canary release (1486066183733), [amp-bind sample](https://ampbyexample.com/components/amp-bind/preview/) gives the following error:
chunk.js:351 Uncaught (in promise) TypeError: Cannot read property 'enqueue' of undefined
    at lb (chunk.js:351)
    at jb (chunk.js:331)
    at chunk.js:72
    at Tb (bind-impl.js:188)
    at Rb (bind-impl.js:137)
    at bind-impl.js:101

@choumx @ericlindley-g

### PR DESCRIPTION
**Pull request by choumx**
Fixes #7329. Use `getExistingServiceForDoc()` to prevent `Chunks` from being installed in each extension that uses it.

If you think it's a good idea, I can follow up this bug fix with a PR to split this file into `chunk.js` and `service/chunk-impl.js` like other services.

### CODE REVIEW

File: src/chunk.js [hunk 1]
```diff
@@ -56,29 +75,6 @@ export function startupChunk(nodeOrAmpDoc, fn) {
...
- * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
- * @param {function(?IdleDeadline)} fn
- * @param {ChunkPriority} priority
- */
-export function chunk(nodeOrAmpDoc, fn, priority) {
-  if (deactivated) {
-    resolved.then(fn);
-    return;
-  }
-  const service = fromClassForDoc(nodeOrAmpDoc, 'chunk', Chunks);
```
Comment (cramforce) [inline src/chunk.js]: Just calling `getExistingServiceForDoc` should have the desired result code size wise. I personally feel that is a nicer pattern that we should move towards.

Comment (choumx) [inline src/chunk.js]: Agreed, that's actually what I'm doing in this PR.

- Extensions use `chunksForDoc` -> `getExistingServiceForDoc`
- Entry points continue to use `startupChunk` -> `fromClassForDoc` to install the service as a side effect

Did you mean I should change `startupChunk` to use `getExistingServiceForDoc` and explicitly install the service in each entry point?

Comment (cramforce) [inline src/chunk.js]: No, I meant that you can keep the `chunk` function to limit the exported interface to just that function.

Comment (choumx) [inline src/chunk.js]: Ohh, I assumed that we'd want to make `Chunks` more consistent with other services. Done!

**Pull request merged**